modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -400,6 +407,79 @@ void createExoplanetSystem(const std::string& starName) {
             );
         }
     }
+
+    // Habitable Zone
+    bool hasTeff = !std::isnan(system.starData.teff);
+    bool hasLuminosity = !std::isnan(system.starData.luminosity);
+
+    if (hasTeff && hasLuminosity) {
+        const glm::vec2 zone = computeHabitableZone(
+            system.starData.teff,
+            system.starData.luminosity
+        );
+
+        float meanInclination = 0.f;
+        for (const ExoplanetDataEntry& p : system.planetsData) {
+            meanInclination += p.i;
+        }
+        meanInclination /= static_cast<float>(system.planetsData.size());
```

Comment:
Could it ever happen that the planetsData.size() is 0 here?  In that case, this would lead to an exception

Reply:
That should not be possible, since a system without planets are not considered. Previously there is a check for this
```
    if (system.planetNames.empty()) {
        LERROR(fmt::format("Exoplanet system '{}' could not be found", starName));
        return;
    }
```
However, I can make it more clear by checking agains `planetsData` instead of `planetNames` (these vectors should always have the same length)